canonical-web-and-design/jaas.ai
# Resize and restyle 'What people saying' section on homepage

## Done

Resize and restyle 'What people saying' section on the homepage

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- View testimonials section on homepage and ensure styling is correct through breakpoints

## Details

Fixes #201
